app/controllers/sessions_controller.rb

```diff
@@ -2,6 +2,11 @@ class SessionsController < ApplicationController
   skip_before_action :request_login
 
   def new
+    if Settings.features.start_page
+      render "pages/start"
```

Comment:
shouldn't this be a redirect?

https://github.com/DFE-Digital/publish-teacher-training/blob/d38cf8b79527183ec2f716a79935ffc9d605e9eb/config/routes.rb#L11

Reply:
It could be but this seemed the simplest way when we were in a hurry the other day. Redirecting would mean another action and/or controller and I'm not sure there's much benefit to be had for that overhead?

Reply:
There is already __the__ action `start` and the controller `page`

https://github.com/DFE-Digital/publish-teacher-training/blob/25164abdbd66bd233f2b36dcb76e0005283df332/config/routes.rb#L203

:confused:

Reply:
I think that this isn't a start page, this is a signin intercept page. For example, if somebody goes to `/signin` they see this screen.